doc/ale.txt

```diff
@@ -946,6 +947,36 @@ ALEWarningSign                                                 *ALEWarningSign*
   The highlight used for warning signs. See |g:ale_set_signs|.
 
 
+-------------------------------------------------------------------------------
+3.2. Options for write-good Linter                     *ale-write-good-options*
+
+The options for the write-good linter are global because it does not make
+sense to have them specified on a per-language basis.
```

Comment:
This explanation doesn't quite sound right. Instead of making excuses for why this is different, explain the advantages of configuring everything in one place instead.